modules/Bio/EnsEMBL/Analysis/Hive/Config/Genome_annotation_conf.pm

```diff
@@ -134,6 +135,9 @@ sub default_options {
 # Pipe and ref db info
 ########################
 
+    'red_logic_name'            => 'repeatdetector', # logic name for the Red repeat finding analysis
+
+    'projection_source_db_name'    => 'homo_sapiens_core_100_38', # This is generally a pre-existing db, like the current human/mouse core for example
```

Comment:
This should not be in the PR

Reply:
I've just removed the projection_source_db_name initialisation as it's also a duplicate.